### PR TITLE
#35 List of routes view

### PR DESCRIPTION
**Pull request by juliaanholcer**
Add some routes via form to see routes on this view.

### CODE REVIEW

**Comment by jacol12345** [comment]
@juliaanholcer You can rebase the PR branch ;)

**Comment by juliaanholcer** [comment]
@jacol12345 working on it :-) There's nothing better than some conflicts early in the morning.

**Comment by juliaanholcer** [comment]
I wanted to make all of this on the computer with windows, and it was a really bad idea...

File: frontend/src/views/RouteFormView.vue [hunk 1]
```diff
@@ -115,18 +117,21 @@ export default {
...
 
     validateForm() {
       this.errors = [];
 
-      Object.keys(this.route).forEach((key) => {
-        if (!this.route[key]) {
-          this.errors.push(`${key} is required`);
-        }
-      });
+      Object.keys(this.route)
```
Comment (jacol12345) [inline frontend/src/views/RouteFormView.vue]: If you want to make it super-nice :D 
```
this.errors = Object.entries(this.route).filter([key, val] => !val).map([key, val] => `${key} is required`);
```

Comment (jacol12345) [reply frontend/src/views/RouteFormView.vue]: No need to change, I was just boasting :)

**Pull request merged**